lab_5_scrapper/scrapper.py

```diff
@@ -204,13 +416,40 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    if not base_path.is_dir():
+        base_path.mkdir(parents=True, exist_ok=True)
+    elif base_path.exists():
+        shutil.rmtree(base_path)
+        base_path.mkdir(parents=True, exist_ok=True)
```

Comment:
it is enough to remove directory first and then create it from scratch

Reply:
Changed it